pkg/node/devnode.go

```diff
@@ -115,7 +116,9 @@ func NewDevBee(logger log.Logger, o *DevOptions) (b *DevBee, err error) {
 	}
 	b.stateStoreCloser = stateStore
 
-	batchStore, err := batchstore.New(stateStore, func(b []byte) error { return nil }, logger)
+	swarmAddress := test.RandomAddress()
```

Comment:
I would suggest not to include the `test` package here and instead create an address by calling one of the `swarm` create address functions.